modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -523,3 +523,63 @@
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/merp.ogg' = 1)
 	young = FALSE
 
+/obj/item/toy/plush/bonnie
+	name = "punko fop blue bunny"
+	desc = "An 'adorable', if big eyed rabbit girl. This one is blue. Produced by the Punko Fop Company<i><b>TM</b></i>. Spits out various Rabbit facts of dubious validity."
+	icon = 'modular_skyrat/master_files/icons/obj/plushes.dmi'
+	icon_state = "plushie_pfbonnie"
+	gender = FEMALE
+	attack_verb_continuous = list("pats", "hugs", "scolds", "pets")
+	attack_verb_simple = list("pat", "hug", "scold", "pet")
+	squeak_override = list('sound/effects/mousesqueek.ogg' = 1)
+ //All lowercase messages are intentional
+	var/static/list/bonhit_response = list("Rabbits are prey animals and are therefore constantly aware of their surroundings.", "Things to jump up on (they like to be in high places)", "become a rabbit today!", "Be cunning and full of tricks...", "Subscription confirmed! Thank you for choosing RABBITFACTS <i><b>TM</b></i>!",)
+
+/obj/item/toy/plush/bonnie/attackby(obj/item/attacking_item, mob/living/user, params)
```

Comment:
```suggestion
/obj/item/toy/plush/punko/attackby(obj/item/attacking_item, mob/living/user, params)
```
Use a parent item here and inherit it. Don't dupe all the code for each one

Reply:
Noted.

Reply:
Though again. How would I do that?

Reply:
Sorted.